Source/Widgets/CesiumInspector/CesiumInspectorViewModel.js

```diff
@@ -499,18 +516,22 @@ define([
 
             that.tile = selectedTile;
 
-            eventHandler.removeInputAction(ScreenSpaceEventType.LEFT_CLICK);
             that.pickTileActive = false;
         }
+
         this._pickTile = createCommand(function() {
             that.pickTileActive = !that.pickTileActive;
+        });
 
-            if (that.pickTileActive) {
+        this.pickTileActive.subscribe(function(val) {
+            if (val) {
                 eventHandler.setInputAction(selectTile, ScreenSpaceEventType.LEFT_CLICK);
             } else {
                 eventHandler.removeInputAction(ScreenSpaceEventType.LEFT_CLICK);
             }
         });
+
+        knockout.track(this);
```

Comment:
Change this back to use the second argument to define which properties we want to convert to observables.  We don't want it to use every property on `this`